spark/src/test/scala/org/apache/spark/sql/delta/metric/IncrementMetricSuite.scala

```diff
@@ -70,7 +72,13 @@ abstract class IncrementMetricSuiteBase extends QueryTest with SharedSparkSessio
     val increment = IncrementMetric(Literal(true), incrementMetric)
     val incrementPreFilterMetric = createMetric(sparkContext, "incrementPreFilter")
     val incrementPreFilter = IncrementMetric(Literal(true), incrementPreFilterMetric)
-    val ifCondition: Expression = ('a < Literal(20)).expr
+    // Implicit `Column.expr` doesn't work due to ambiguity
```

Comment:
You could also avoid importing one of these methods. In general RichColumn is a bit better because it is bound to session.